docs/debugging/inspect/main.go

```diff
@@ -19,125 +19,95 @@ package main
 
 import (
 	"bufio"
-	"encoding/binary"
-	"encoding/hex"
-	"encoding/json"
-	"errors"
 	"flag"
 	"fmt"
-	"hash/crc32"
-	"io"
 	"io/ioutil"
-	"log"
 	"os"
 	"strings"
-
-	"github.com/secure-io/sio-go"
+	"time"
```

Comment:
This doesn't appear to be buildable any more?

Is something missing?

Reply:
> This doesn't appear to be buildable any more?

Yes, some files weren't commited, I fixed .gitignore for that